### ISSUE TITLE
Unable to do `npm install` on OSX with node version 4.x

### ISSUE THREAD

**Issue by dopeddude** [mcavage/node-bunyan-syslog#31]
$ npm install bunyan-syslog --save

```
> bunyan-syslog@0.3.0 install /Users/ashwinikumar/work/1_hs/hotelsoftrzr/node_modules/bunyan-syslog
> node-gyp rebuild

  CXX(target) Release/obj.target/syslog/src/syslog.o
In file included from ../src/syslog.cc:7:
In file included from ../../nan/nan.h:74:
In file included from ../../nan/nan_new.h:190:
../../nan/nan_implementation_12_inl.h:181:66: error: too many arguments to function call, expected at most 2, have 4
  return v8::Signature::New(v8::Isolate::GetCurrent(), receiver, argc, argv);
         ~~~~~~~~~~~~~~~~~~                                      ^~~~~~~~~~
/Users/ashwinikumar/.node-gyp/4.2.1/include/node/v8.h:4675:3: note: 'New' declared here
  static Local<Signature> New(
  ^
In file included from ../src/syslog.cc:7:
../../nan/nan.h:165:25: error: redefinition of '_NanEnsureLocal'
NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Local<T> val) {
                        ^
../../nan/nan.h:160:25: note: previous definition is here
NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Handle<T> val) {
../../nan/nan.h:310:12: error: no matching function for call to '_NanEnsureLocal'
    return NanEscapeScope(NanNew(v8::Undefined(v8::Isolate::GetCurrent())));
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../../nan/nan.h:296:43: note: expanded from macro 'NanEscapeScope'
# define NanEscapeScope(val) scope.Escape(_NanEnsureLocal(val))
                                          ^~~~~~~~~~~~~~~
../../nan/nan.h:165:25: note: candidate template ignored: substitution failure [with T = v8::Primitive]
NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Local<T> val) {
```

$ npm version

```
{ npm: '3.3.8',
  ares: '1.10.1-DEV',
  http_parser: '2.5.0',
  icu: '56.1',
  modules: '46',
  node: '4.2.1',
  openssl: '1.0.2d',
  uv: '1.7.5',
  v8: '4.5.103.35',
  zlib: '1.2.8' }
```

**Comment by koresar** [comment]
It's not OS X issue, but node. The node v4 is not supported yet.

**Comment by dopeddude** [comment]
@koresar when should I expect the support of node v4.
.
And which node version should I downgrade to get it going.

**Comment by koresar** [comment]
@dopeddude look around :)
there is a PR for that: https://github.com/mcavage/node-bunyan-syslog/pull/30

I'd recommend forking this: https://github.com/sempasha/node-bunyan-syslog

**Comment by dopeddude** [comment]
@koresar Thanks, will give it a try. Thanks again.

**Comment by pfmooney** [comment]
This should be fixed in v0.3.1
